src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -1465,6 +1484,98 @@ private static BundleEntryComponent explanationOfBenefit(BundleEntryComponent pe
+   * Subject:	1..1	Reference(Patient | Group)	Individual service was done for/to
+   * Performer - Actor: 	1..1	Reference Individual who was performing
+   */
+  private static BundleEntryComponent chargeItem(RandomNumberGenerator rand,
+          BundleEntryComponent personEntry, Bundle bundle, BundleEntryComponent encounterEntry,
+          HealthRecord.Entry chargeItem) {
+
+    ChargeItem chargeItemResource = new ChargeItem();
+
+    final String STRUCTURE_DEFINITION = "https://verily-src.github.io/verily-fhir-ig/StructureDefinition/verily-charge-item";
+    final String ENRICHED_REVENUE_CODES = "https://verily-src.github.io/verily-fhir-ig/StructureDefinition/enriched-nubc-revenue-codes";
+    final String REVENUE_CATEGORY = "https://verily-src.github.io/vhp-hds-vvs-fhir-ig/StructureDefinition/nubc-revenue-category";
+    final String ENRICHED_CPT = "https://verily-src.github.io/verily-fhir-ig/StructureDefinition/enriched-cpt";
+    final String ENRICHED_HCPCS = "https://verily-src.github.io/vhp-hds-vvs-fhir-ig/StructureDefinition/enriched-hcpcs";
+    final String ENRICHED_NDC = "";
+    final String TRANSACTION_TYPE = "";
+    final String DETAILED_COST = "https://verily-src.github.io/vhp-hds-vvs-fhir-ig/StructureDefinition/detailed-cost";
+    final String CHARGE_AMOUNT = "";
+    final String REVENUE_CODES = "https://www.nubc.org/CodeSystem/RevenueCodes";
+    final String HCPCS_CODES = "http://terminology.hl7.org/CodeSystem/HCPCS";
+
+          
+    // Profile 
+    if (USE_US_CORE_IG) {
+      // The metadata about a resource. This is content in the resource that is maintained by the infrastructure. 
+      // Changes to the content might not always be associated with version changes to the resource.
+      Meta meta = new Meta();
+      meta.addProfile("http://hl7.org/fhir/us/core/StructureDefinition/us-core-condition");
+      chargeItemResource.setMeta(meta);
+    } 
+    
+    // Status 
+    chargeItemResource.setStatus(ChargeItemStatus.BILLABLE);
+
+    // code
+    chargeItemResource.setCode(new CodeableConcept()
+                      .addCoding(new Coding("System",
+                                            "01510",
+                                            "Some item")));
+
+    // References
+    chargeItemResource.setSubject(new Reference(personEntry.getFullUrl()));
+    chargeItemResource.setContext(new Reference(encounterEntry.getFullUrl()));
+
+    // enrichedRevenueCode
+    Extension enrichedRevenueCode = codeEnrichment(ENRICHED_REVENUE_CODES, REVENUE_CODES);
+    chargeItemResource.addExtension(enrichedRevenueCode);
+
+    // enrichedHCPCS
+    Extension enrichedHCPCS = codeEnrichment(ENRICHED_HCPCS, HCPCS_CODES);
+    chargeItemResource.addExtension(enrichedHCPCS);
+
+    // detailedCost
+    Extension moneyUrl = new Extension(DETAILED_COST);
+    Extension moneyExtension = new Extension("totalCost");
+    Money moneyResource = new Money();
+    moneyResource.setValue(4928.71);
```

Comment:
Is this always supposed to constant?

Reply:
No. This falls under the same question you asked above about the randomize fields. I made the same comment a few days ago; At this stage, the values are all constant in order to test functionality, but we should review if it is important to make them randomized and _how_ could we do It. I mentioned this before, so I'll proceed to add a GDoc explaining possible solutions to be explored